### ISSUE TITLE
Add space in directives not allowed

### ISSUE THREAD

**Issue by ihsanturk** [koalaman/shellcheck#1940]
- Rule Id: SC1073
- My shellcheck version: 0.7.1
- [X] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC1073)
- [ ] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh
# shellcheck source=/Users/username/Library/Application Support/somedir/somefile.sh
. "$HOME/Library/Application Support/somedir/somefile.sh"
```

#### Here's what shellcheck currently says:
`# shellcheck source=/Users/username/Library/Application Support/somedir/somefile.sh`
^-- SC1073: Couldn't parse this shellcheck directive. Fix to allow more checks.
                                                            ^-- SC1072: Expected '=' after directive key. Fix any mentioned problems and try again.

#### Here's what I wanted or expected to see:
Maybe I could use quotes or backslash with source files in directives to escape whitespace.

shellcheck source=`"`/Users/username/Library/Application Support/somedir/somefile.sh`"`
shellcheck source=/Users/username/Library/Application`\` Support/somedir/somefile.sh
